baybe/recommenders/pure/bayesian/base.py

```diff
@@ -35,39 +27,27 @@ class BayesianRecommender(PureRecommender, ABC):
     surrogate_model: Surrogate = field(factory=GaussianProcessSurrogate)
     """The used surrogate model."""
 
-    acquisition_function_cls: Literal[
-        "PM", "PI", "EI", "UCB", "qPI", "qEI", "qUCB", "VarUCB", "qVarUCB"
-    ] = field(default="qEI")
+    acqf: AcquisitionFunction = field(
```

Comment:
I'd prefer the full name here (see my other comment)

Reply:
i dislike the very lengthy variable name which gets even wors when combined with more attributes (eg `botorch_acquisition_function_cls` or so), so @AdrianSosic must decide

Reply:
Puh, this really puts me into a dilemma :D Since a while, I generally prefer the mantra "explicit names, no abbreviations" because it simply helps in understanding the code. While "acqf" is not terrible here, I think I still have a slight tendency to the full explicit name, just like @AVHopp ... But this time I'm playing the ball back to you, since it has become your PR 😄

Reply:
ok I will change it to long name everywhere then

I also scold you for the inconsistency tho as clearly you've named this branch `acqfs` ;)

Reply:
pushed the follwoing changes now:
- the baybe acquisition function (which is a user facing keyword) is called `acquisition_function` as per your request
- I made in some cases a more explicit difference between botorch and baybe acquisition function, espeically int he import I now call a botorch variant `BotorchAcquisitionFunction` instead of `AcquisitionFunction` which remains reserved for baybe acqfs
- Additionally, for botorch attributes that are part of our code I remain calling it `acqf`. This is consistent with botorch functions named `optimize_*_acqf` NAD is not userfacxing unlike the other keyword
- `_acqf` is now called `_botorch_acqf`
- `_setup_acqf` is now called `_setup_botorch_acqf`

if you want check the last commits and resolve this

Reply:
Sounds very reasonable to me 👍🏼  I'll leave it to @AVHopp to resolve.